ocs_ci/utility/perf_dash/dashboard_api.py

```diff
@@ -0,0 +1,510 @@
+        """
+        Query of the ID of test name in the DB
+
+        Args:
+            test (str): The test name (e.g. FIO)
+
+        Returns:
+             int : the Test ID in the DB
+
+        """
+        return self.get_id_by_name("tests", test)
+
+    def get_platform_id(self, platform=None):
+        """
+        Query of the ID of platform name in the DB
+
+        Args:
+            platform (str): The platform name (e.g. AWS)
+
+        Returns:
+             int : the Platform ID in the DB
+
+        """
+        return self.get_id_by_name("platform", platform)
+
+    def get_topology_id(self, topology=None):
+        """
+        Query of the ID of platform name in the DB
+
+        Args:
+            topology (str): The Topology name (e.g. 3-AZ)
+
+        Returns:
+             int : the Topology ID in the DB
+
+        """
+        return self.get_id_by_name("az_topology", topology)
+
+    def get_version_name(self, version=0):
+        """
+        Query of the Name of version ID in the DB
+
+        Args:
+            version (int): The version ID
+
+        Returns:
+             str : the version name in the DB
+
+        """
+        return self.get_name_by_id("versions", version)
+
+    def get_test_name(self, test=0):
+        """
+        Query of the Name of test ID in the DB
+
+        Args:
+            test (int): The test ID
+
+        Returns:
+             str : the Test Name in the DB
+
+        """
+        return self.get_name_by_id("tests", test)
+
+    def get_platform_name(self, platform=0):
+        """
+        Query of the IName of platform ID in the DB
+
+        Args:
+            platform (int): The platform ID
+
+        Returns:
+             str : the Platform Name in the DB
+
+        """
+        return self.get_name_by_id("platform", platform)
+
+    def get_topology_name(self, topology=0):
+        """
+        Query of the Name of platform ID in the DB
+
+        Args:
+            topology (int): The Topology ID
+
+        Returns:
+             str : the Topology Name in the DB
+
+        """
+        return self.get_name_by_id("az_topology", topology)
+
+    def get_version_builds(self, version=None):
+        """
+        Query the list of build in specific version
+
+        Args:
+            version (str): The version name (e.g. 4.9.0)
+
+        Returns:
+             dict : dictionary of (Name: ID), None if not exist
+
+        """
+        ver_id = self.get_version_id(version)
+        if ver_id is None:
+            return None
+
+        results = {}
+        query = f"SELECT id, name FROM builds WHERE version = {ver_id} ;"
+        if self._run_query(query=query):
+            for (build_id, name) in self.cursor:
+                results[name] = build_id
+
+        return None if results == {} else results
+
+    def get_build_id(self, version, build):
+        """
+        Getting the build ID for specific build of version.
+        if the build is not exist, return None
```

Comment:
spelling suggestion: if the build does not exist , return None

Reply:
done.